docs/docs/themes/using-multiple-gatsby-themes.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Using Multiple Gatsby Themes
+título: Usando de múltiples temas de Gatsby
```

Comment:
```suggestion
título: Usar múltiples temas de Gatsby
```

Reply:
Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.